src/Glpi/Console/Application.php

```diff
@@ -567,15 +576,10 @@ function ($path) {
         return true;
     }
 
-    public function extractNamespace(string $name, ?int $limit = null): string
```

Comment:
The `extractNamespace` method was used to force grouping of plugins commands. It should not be removed.

![image](https://github.com/glpi-project/glpi/assets/33253653/0447fa79-49c0-42c8-a0f1-8cce4005a7ce)